tests/test_getRaw.py

```diff
@@ -33,16 +33,15 @@
 
 import lsst.pex.exceptions as pexExcept
 import lsst.daf.persistence as dafPersist
-from lsst.obs.base import MakeRawVisitInfo
 from lsst.afw.image import RotType
-from lsst.afw.coord import IcrsCoord, Coord
-from lsst.afw.geom import degrees
+from lsst.afw.geom import degrees, SpherePoint
 
 
 # Desired VisitInfo values for visit 229388, shared between test_getRaw.py and
 # test_getInstcal.py
 
-boresightRaDec = IcrsCoord('02:51:16.790', '-00:00:05.699')
+boresightRaDec = SpherePoint(42.81995833, -0.00158305, degrees)
+# boresightRaDec = IcrsCoord('02:51:16.790', '-00:00:05.699')
```

Comment:
Why did you include this line?  If you want to preserve the coordinates I don't think it makes sense to put them in a IcrsCoord.

Reply:
Agreed. I'll delete it. I was intending to show the sexagesimal representation of the original, but I dooubt it's worth doing, and I should have done it in a less confusing way!